docs/sql-ref-syntax-qry-select.md

```diff
@@ -151,6 +151,18 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { named_expression [ , ... ] }
 
      Specifies aliases for one or more source window specifications. The source window specifications can
      be referenced in the widow definitions in the query.
+     
+* **regex_column_names**
+
+     When `spark.sql.parser.quotedRegexColumnNames` is true, quoted identifiers (using backticks) in `SELECT`
+     statement are interpreted as regular expressions and `SELECT` statement can take regex-based column specification.
+     For example, below SQL will only take column `c`:
+
+     ```sql
+     SELECT `(a|b)?+.+` FROM (
+       SELECT 1 as a, 2 as b, 3 as c
+     ) src
```

Comment:
Do we need this `src`? I think we can omit this.

Reply:
```
spark-sql> SELECT a FROM (SELECT 1 as a, 2 as b, 3 as c);
1
```

Reply:
> Do we need this `src`? I think we can omit this.

Ur, not need, Omit this.